FGB-insite-team-task/src/app/fgb-details/fgb-dropdown-list/fgb-dropdown-list.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="list-container">
+    <div class="container">
+        <div class="row">
+            <div class="col-sm">
+                GAME:
+            </div>
+            <div class="col-sm">
+                <select class="dropdown" (change)="gameSelected($event.target.value)">
+                    <option>
+                    </option>
+                    <option>
+                        0
+                    </option>
+                    <option *ngFor="let game of service.gameList" [value]="game.id">
+                        {{game.description}}
+                    </option>
+                </select>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-sm">
+                SEASON:
+            </div>
+            <div class="col-sm">
+                <select class="dropdown" (change)="seasonSelected($event.target.value)">
```

Comment:
In angular you don't need to implement a `(change)` event handler just to set the value of the dropdown - this can be done using [two-way binding and the `[(ngModel)]` directive](https://angular.io/guide/built-in-directives#ngmodel-two-way-binding).